src/repositories/request-repository.ts

```diff
@@ -221,7 +225,7 @@ export class RequestRepository {
         pinned: fields.pinned,
         updatedAt: updatedAt,
       })
-      .whereIn('id', ids)
+      .where('id', 'in', connection.raw('SELECT * FROM UNNEST (?::uuid[])', [ids]))
```

Comment:
I don't really understand what this is doing well enough to comment on the pros/cons of this approach.  Are there any downsides to this type of update vs a normal update?  Why does this get around the limitation?  Could this wind up getting limited in the same way in a future release of postgres?

Just looks like of weird and brittle to me, like we're relying on an implementation detail to hack our way around how postgres wants to be used.